third-party/easyloggingpp/src/easylogging++.h

```diff
@@ -2733,10 +2771,11 @@ class AsyncDispatchWorker : public base::IWorker, public base::threading::Thread
   virtual ~AsyncDispatchWorker();
 
   bool clean(void);
-  void emptyQueue(void);
+  void emptyLogQueue(void);
```

Comment:
Renaming may apply to public API and complicate future rebase -is it necessary?

Reply:
Renaming removed - done